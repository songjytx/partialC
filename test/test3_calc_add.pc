void main(){
	int a = 4 * ( 1 + 2 );
	printi(a);
	float b = 1.2 + 3.5;
	printf(b);
}