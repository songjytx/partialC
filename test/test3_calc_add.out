12
4.700000
